core/storage/suggestion/gae_models.py

```diff
@@ -1990,3 +2000,1275 @@ def export_data(
                     model.last_contribution_date.isoformat())
             }
         return user_data
+
+
+class TranslationSubmitterTotalContributionStatsModel(base_models.BaseModel):
+    """Records the Total Translation contribution stats and data of
+    recent_review keyed per (contributor_id, language_code) tuple.
+    Its IDs will be in the following
+        structure: [contributor_id][language_code]
```

Comment:
This is not quite right, it doesn't match your implementation.

`The ID of each model is formatted as [language_code].[contributor_id]`

Please make sure other docstrings are fully correct as well, otherwise we go through unnecessary review cycles.

Reply:
Updated